en/mulesoft/modules/ROOT/pages/ms_composer_flows.adoc

```diff
@@ -480,7 +480,7 @@ When mapping source data fields:
 To map source data fields to target fields:
 
 . From the *Composer Flows* tab, open a flow and click *Edit*.
-. In an action card, in a field that supports *Pick from List*, select *Picklist Mapping* from the menu. +
+. In an action card, in a field that supports *Pick from List* or any `String` field, select *Picklist Mapping* from the menu. +
```

Comment:
Is String a data type here or a field? If the former, I think our style is no font change. If the latter, then bold.
https://confluence.internal.salesforce.com/display/MTDT/MuleSoft+CX+Writing+Style+Reference#MuleSoftCXWritingStyleReference-fontchoices